boilerplate/app/components/EmptyState.tsx

```diff
@@ -108,22 +110,28 @@ interface EmptyStatePresetItem {
   button: TextProps["text"]
 }
 
-const EmptyStatePresets = {
-  generic: {
-    imageSource: sadFace,
-    heading: translate("emptyStateComponent.generic.heading"),
-    content: translate("emptyStateComponent.generic.content"),
-    button: translate("emptyStateComponent.generic.button"),
-  } as EmptyStatePresetItem,
-} as const
-
 /**
  * A component to use when there is no data to display. It can be utilized to direct the user what to do next.
  * @see [Documentation and Examples]{@link https://docs.infinite.red/ignite-cli/boilerplate/components/EmptyState/}
  * @param {EmptyStateProps} props - The props for the `EmptyState` component.
  * @returns {JSX.Element} The rendered `EmptyState` component.
  */
 export function EmptyState(props: EmptyStateProps) {
+  const {
+    themeContext,
+    themed,
+    theme: { spacing },
+  } = useAppTheme()
+
+  const EmptyStatePresets = {
+    generic: {
+      imageSource: themeContext === "dark" ? sadFaceDark : sadFace,
```

Comment:
Good catch! I think we should probably also do this for the face on the Welcome screen.

I wonder if we can add this as a general image pattern where the swapping can be handled automatically? not necessary, just if you want

Reply:
While fixing the welcome screen, i realized i can just use `tintColor` on these images since there's hardly any color data. In dark mode, just tint the images white! It converts all non transparent pixels to that color. I'll remove some of the image assets i added to this PR!

```jsx
<Image
  style={$welcomeFace}
  source={welcomeFace}
  resizeMode="contain"
  tintColor={themeContext === "dark" ? "#FFFFFF" : undefined}
/>
```

Reply:
Here's what the welcome screen looks like now: 
![2EF7AEFF-E099-40C6-ACF3-34BEE7592E17-52253-00023A4B247E5E68](https://github.com/user-attachments/assets/d7dde65b-84f5-4aba-99da-38dc721feb4e)